mycli/main.py

```diff
@@ -557,12 +557,15 @@ def one_iteration(document=None):
                 logger.debug("connection id to kill: %r", connection_id_to_kill)
                 # Restart connection to the database
                 sqlexecute.connect()
-                for title, cur, headers, status in sqlexecute.run('kill %s' % connection_id_to_kill):
-                    status_str = str(status).lower()
-                    if status_str.find('ok') > -1:
-                        logger.debug("cancelled query, connection id: %r, sql: %r",
-                                     connection_id_to_kill, document.text)
-                self.output("cancelled query", err=True, fg='red')
+                try:
+                    for title, cur, headers, status in sqlexecute.run('kill %s' % connection_id_to_kill):
+                        status_str = str(status).lower()
+                        if status_str.find('ok') > -1:
+                            logger.debug("cancelled query, connection id: %r, sql: %r",
+                                         connection_id_to_kill, document.text)
+                            self.output("cancelled query", err=True, fg='red')
+                except Exception as e:
+                    self.output("cancel query with error: %r" % e, err=True, fg='red')
```

Comment:
Can you change the sentence to `Encountered error while canceling query: %r`

Otherwise, it looks great.